doc/sphinx/addendum/rewrite-rules.rst

```diff
@@ -0,0 +1,192 @@
+.. _rewrite_rules:
+
+User-defined rewrite rules
+==========================
+
+.. warning::
+
+   Rewrite rules are highly experimental.
+
+   In particular, ill-typed rewrite rules will lead to mistyped expressions,
+   and manipulating these will most often result in anomalies.
```

Comment:
Also, IIUC, users can define unsound rewrite rules that allow a proof of False (similar to the risks of using `Axiom`).

Reply:
They definitely can (they can simply write `Symbol f : False.`), but being able to write mistyped expressions is far worse than breaking consistency : Coq can work fine with an inconsistent system (with a few limitations), but it will stop almost immediately when it encounters a mistyped expression.

Reply:
I was implicitly suggesting that you add a sentence mentioning this, since correctness is the raison d'être for Coq.